sktime/annotation/base/_base_panel.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+"""
+Base class template for panel annotator scitype
+
+    class name: BasePanelAnnotator
+
+Scitype defining methods:
+    fitting              - fit(self, X, Y=None, Z=None)
+    annotating           - predict(self, X, Z=None)
+
+    NOT YET IMPLEMENTED
+    updating (minibatch) - update(self, X, Y=None, Z=None)
+    update&annotate      - update_predict(self, X, X_new, Y=None, Z=None, Z_new=None)
+
+Inspection methods:
+    hyper-parameter inspection  - get_params()
+    fitted parameter inspection - get_fitted_params()
+
+State:
+    fitted model/strategy   - by convention, any attributes ending in "_"
+    fitted state flag       - check_is_fitted()
+
+copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""
+
+__author__ = ["fkiraly"]
+
+import pandas as pd
+
+from sktime.base import BaseEstimator
+
+
+class BasePanelAnnotator(BaseEstimator):
+    """Base annotator
+
+    The base annotator specifies the methods and method
+    signatures that all annotators have to implement.
+
+    Specific implementations of these methods is deferred to concrete
+    annotators.
+    """
+
+    # default tags
+    _tags = {
+        "handles-panel": True,  # can handle panel annotations, i.e., list X/y?
```

Comment:
This tag isn't in the registry? I guess it would always be true anyway for panel annotator?